refractr/cfg.py

```diff
@@ -144,25 +144,24 @@ def ECR_REPOURL(self):
     def IMAGE_NAME_AND_TAG(self):
         return self('IMAGE_NAME_AND_TAG', f'{self.ECR_REPOURL}:{self.VERSION}')
 
+    # GITHUB Actions env vars
+    # https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
     @property
     @lru_cache()
-    def TRAVIS(self):
-        return self('TRAVIS', False)
+    def GITHUB_REF(self):
+        return self('GITHUB_REF', '')
```

Comment:
Any particular reason you're setting this to empty string instead of False?

Reply:
`GITHUB_REF` value use to be something like `"refs/<tags|heads>/"`, and I keep empty string to follow the same type in this case. The usage in some comparisons will end up the same way.